pkg/converters/ingress/defaults.go

```diff
@@ -95,24 +95,26 @@ func createDefaults() map[string]string {
 		types.GlobalModsecurityTimeoutIdle:       "30s",
 		types.GlobalModsecurityTimeoutProcessing: "1s",
 		types.GlobalModsecurityTimeoutServer:     "5s",
-		types.GlobalNbprocBalance:                "1",
-		types.GlobalNoTLSRedirectLocations:       "/.well-known/acme-challenge",
-		types.GlobalPathTypeOrder:                "exact,prefix,begin,regex",
-		types.GlobalRedirectFromCode:             "302",
-		types.GlobalRedirectToCode:               "302",
-		types.GlobalSSLDHDefaultMaxSize:          "2048",
-		types.GlobalSSLHeadersPrefix:             "X-SSL",
-		types.GlobalSSLOptions:                   defaultSSLOptions,
-		types.GlobalStatsPort:                    "1936",
-		types.GlobalSyslogFormat:                 "rfc5424",
-		types.GlobalSyslogLength:                 "1024",
-		types.GlobalSyslogTag:                    "ingress",
-		types.GlobalTimeoutClient:                "50s",
-		types.GlobalTimeoutClientFin:             "50s",
-		types.GlobalTimeoutStop:                  "10m",
-		types.GlobalUseCPUMap:                    "true",
-		types.GlobalUseForwardedProto:            "true",
-		types.GlobalUseHTX:                       "true",
-		types.GlobalDefaultBackendRedirectCode:   "302",
+		// Ref: https://github.com/haproxy/spoa-modsecurity/blob/3c895f3e7dd291dba19d57ba054b277e6fb80ca4/README#L70
+		types.GlobalModsecurityArgs:            "unique-id method path query req.ver req.hdrs_bin req.body_size req.body",
```

Comment:
Move the comment to the same line of the declaration. Yep it'll be a bit long.

Reply:
Sorry didn't ask before, please make it alphabetic order here as well.

Reply:
I had already read your mind :)

https://github.com/jcmoraisjr/haproxy-ingress/pull/948/files#diff-8f9b52a4ac79f8a335f9d75297a55d8061661ff11103c8d78ec9472e87dead7c